fonts.css

```diff
@@ -0,0 +1,13 @@
+@font-face {
+	font-family: "GothamPro";
+	src: url("../fonts/GothamPro.woff") format("woff"),
+		url("fonts/GothamPro.woff2") format("woff2");
+	font-weight: normal;
+}
+
+@font-face {
+	font-family: "GothamPro-Bold";
+	src: url("../fonts/GothamPro-Bold.woff") format("woff"),
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/PolinaYartseva_axion/pullRequest?prid=5542092)